Python/Product/PythonTools/PythonTools/Project/ImportWizard/ImportSettings.cs

```diff
@@ -454,7 +454,6 @@ InterpreterConfiguration config
                 new Dictionary<string, string> {
                     { MSBuildConstants.IdKey, Path.GetFileName(sourcePath) },
                     { MSBuildConstants.DescriptionKey, config.Description },
-                    { MSBuildConstants.BaseInterpreterKey, config.Id },
```

Comment:
Yep, that's correct. We don't require a base interpreter for virtual environments anymore.